tuf-spec.md

```diff
@@ -822,23 +825,27 @@ repo](https://github.com/theupdateframework/specification/issues).
    TARGETPATH.  The application may use this information to guide download
    decisions.
 
-   DELEGATIONS is an object whose format is the following:
+   "keys_for_delegations" lists the public keys to verify signatures of
+   delegated targets roles. Revocation and replacement of delegated targets
+   roles keys is done by changing the keys in this field in the delegating
+   role's metadata.
 
-       { "keys" : {
-             KEYID : KEY,
-             ... },
-         "roles" : [{
-             "name": ROLENAME,
-             "keyids" : [ KEYID, ... ] ,
-             "threshold" : THRESHOLD,
-             ("path_hash_prefixes" : [ HEX_DIGEST, ... ] |
-              "paths" : [ PATHPATTERN, ... ]),
-             "terminating": TERMINATING,
-         }, ... ]
-       }
+   "delegations" is a list of DELEGATION objects whose format is the following:
 
-   ROLENAME is the name of the delegated role.  For example,
-   "projects".
+        {
+          "name": DELEGATION_NAME,
+          ("path_hash_prefixes" : [ HEX_DIGEST, ... ] |
+          "paths" : [ PATHPATTERN, ... ]),
```

Comment:
TAP 3 only mentions `paths`, but I guess we still want to support `path_hash_prefixes`?